app/controllers/products_controller.rb

```diff
@@ -17,13 +17,10 @@ def index
         results = search_for_products(20)
         @products = ProductDecorator.decorate_collection(results)
       end
-      format.csv do
+      format.xlsx do
         authorize Product, :export?
 
-        results = search_for_products.includes(:investigations)
-        @products = ProductDecorator.decorate_collection(results)
-
-        render csv: @products, filename: "products"
+        @products = search_for_products(20, %i[investigations corrective_actions risk_assessments test_results]).sort
```

Comment:
I think we would want to do something like this to potentially speed up the rendering of the risk assessment `assessed_by`.

```suggestion
        @products = search_for_products(20, [:investigations, :corrective_actions, risk_assessments: [:assessed_by_business, :assessed_by_team], :test_results]).sort
```
It would be worth going through each column and their rendering methods to identify other speed improvements. We have had issues with the database query timing out on the investigations export Excel before. Given there are about as many products as cases and the fact more data is rendered on this endpoint, the performance is likely to be a lot worse so we should try to optimise the query as much as possible.